test/integration/express/app/app.js

```diff
@@ -2,14 +2,20 @@ const axios = require('axios');
 const express = require('express');
 const bodyParser = require('body-parser');
 const app = express();
+require('log-timestamp');
 
 // parse application/x-www-form-urlencoded
 app.use(bodyParser.urlencoded({ extended: false }));
 
 // parse application/json
 app.use(bodyParser.json());
 
+app.get('/', async (req, res) => {
+  res.send("server is ready").status(200);
+});
+
 app.get('/invoke-requests', async (req, res) => {
+  axios.defaults.adapter = require('axios/lib/adapters/http') //handle cross-origin error https://stackoverflow.com/questions/51054286/cross-origin-http-request-originating-from-server-side-nodejs-axios-jsdom
```

Comment:
@shanishiri what is the issue we are working around here? I am skeptical of using something like this in our test apps because it looks like the kind of thing that may significantly change the behavior of the instrumentation and produced data.